spec/cypress/app_commands/scenarios/school_participants.rb

```diff
@@ -18,9 +18,31 @@
 coordinator = FactoryBot.create(:user, :induction_coordinator, email: "school-leader@example.com", full_name: "Ms School Leader")
 coordinator.induction_coordinator_profile.schools = [school]
 
-mentor = FactoryBot.create(:user, :mentor, full_name: "Abdul Mentor", id: "51223b41-a562-4d94-b50c-0ce59a8bb34d", school_cohort: school_cohort)
+mentor_profile = FactoryBot.create(
+  :participant_profile, :mentor,
```

Comment:
Can you split these down on to new lines please? I find this really hard to read